### ISSUE TITLE
Angular and Svelte style properties are not transformed

### ISSUE THREAD

**Issue by CKGrafico** [BuilderIO/mitosis#544]
### I am interested in helping provide a fix!

No

### Which generators are impacted?

- [ ] All
- [X] Angular
- [ ] HTML
- [ ] Qwik
- [ ] React
- [ ] React-Native
- [ ] Solid
- [ ] Stencil
- [X] Svelte
- [ ] Vue
- [ ] Web components

### Reproduction case

https://codesandbox.io/s/hungry-cache-trlkvp

### Expected Behaviour

in angular `[style]="obj"` should be `[ngStyle]="obj"` and in svelte is more complex https://svelte.dev/tutorial/style-directive

### Actual Behaviour

not doing anything

### Additional Information

not sure if you want to cover this in mitosis

**Comment by CKGrafico** [comment]
Sorry to many days without creating issues in your repo 😅 I’m on my phone but should be something like this.

https://mitosis.builder.io/?outputTab=react&code=import+%7B+useStore+%7D+from+%22%40builder.io%2Fmitosis%22%3B%0A%0Aexport+default+function+MyComponent%28props%29+%7B%0A++const+state+%3D+useStore%28%7B%0A++++name%3A+%22Steve%22%2C%0A++++customStyles%3A%0A++++%7B+color%3A+%27red%27%7D%0A++%7D%29%3B%0A++%0A++return+%28%0A++++%3Cdiv+styles%3D%7Bstate.customStyles%7D%3E%0A++++++%3Cinput%0A++++++++css%3D%7B%7B%0A++++++++++color%3A+%22red%22%2C%0A++++++++%7D%7D%0A++++++++value%3D%7Bstate.name%7D%0A++++++++onChange%3D%7B%28event%29+%3D%3E+%7B+state.name+%3D+event.target.value+%7D%7D%0A++++++%2F%3E%0A++++++Hello%0A++++++%7Bstate.name%7D%21+I+can+run+in+React%2C+Vue%2C+Solid%2C+or+Liquid%21%0A++++%3C%2Fdiv%3E%0A++%29%3B%0A%7D


basically you’re doing nothing (as usual in any property) but the property `style` has some special scenarios as I’ve explained above.

**Comment by samijaber** [comment]
- For Angular: it should be a 1-line addition to this map to map `style` to `ngStyle`:

https://github.com/BuilderIO/mitosis/blob/d2635b3cb325f49a827749cfb93fef518cdc2c03/packages/core/src/generators/angular.ts#L61-L65

- For Svelte: one possible workaround would be to inject this styling helper whenever needed:
https://svelte.dev/repl/c7787db098e6495888036598586ec4f3?version=3.31.0

```svelte
<script>
	function styling (node, vars) {
		Object.entries(vars).forEach(([ p, v ]) => { node.style[p] = v })
	}
</script>

<div use:styling={{ 'background-color': 'blue', color: 'white', padding: '12px' }}>
	Hi
</div>
```

low-overhead and will help bring Svelte on par with the other frameworks.

**Comment by CKGrafico** [comment]
Hello this is open but there are merges, any update?

**Comment by samijaber** [comment]
Yup, Forgot to close it! Should be fixed by #559
